app/src/components/Search/PokemonList.js

```diff
@@ -12,7 +12,7 @@ export const PokemonList = connectInfiniteHits(
     });
 
     return (
-      <div>
+      <div className="flex flex-wrap p-1 bg-gray-800">
```

Comment:
I guess we should add `h-full overflow-y-auto` in order for the gray area to fill its parent :)
But then we also need to add `items-start` so that the "triggers" wont stretch to the bottom of the container